pkg/pubsub/pubsub_test.go

```diff
@@ -0,0 +1,168 @@
+// +build integration
+
+// This is an integration test file for pubsub.go. Pubsub simulator needs to be running.
+// Run this using go test -tags=integration
+package pubsub_test
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/joincivil/civil-events-crawler/pkg/model"
+	"github.com/joincivil/civil-events-crawler/pkg/pubsub"
+	"github.com/joincivil/go-common/pkg/generated/contract"
+	ctime "github.com/joincivil/go-common/pkg/time"
+	"math/big"
+	"os"
+	"testing"
+	"time"
+)
+
+const (
+	topicName = "testTopic"
+	subName   = "testSubscription"
+	projectID = "civil-media"
+)
+
+func setupCrawlerPubSub(t *testing.T) *pubsub.CrawlerPubSub {
+	os.Setenv("PUBSUB_EMULATOR_HOST", "localhost:8042")
```

Comment:
👍 This is smarter approach to doing this, will have to do this for the other tests.  I was just passing in the env var in the Makefile command for tests. 🤦‍♂️ There will likely never be a case where this simulator be located anywhere else except local host on that port.